src/Take.Blip.Builder/FlowManager.cs

```diff
@@ -712,6 +728,20 @@ private async Task<bool> ValidateInputAsync(Message message, State state, LazyIn
             return true;
         }
 
+        private (bool, Message) HandleMessage(Message message)
```

Comment:
Suggestion: add field names to this tuple in order to better communicate their semantic. For instance 
`(bool MessageHasChanged, Message NewMessage)`